assets/scripts/lobby/chat.js

```diff
@@ -454,11 +454,11 @@ function addToRoomChat(data) {
 }
 
 function isPlayerMuted(username) {
-  if (mutedPlayers.indexOf(username) !== -1) {
-    return true;
+  if (!username) {
+    return false;
```

Comment:
Did you need to do this?

Reply:
theres a few sections where the username being passed in is undefined (i.e. when its system messages etc). I can update each one of those areas individually but feel its better to do a sanity check here